Source/Examples/PropertyGrid/SimpleDemo/MainWindow.xaml

```diff
@@ -2,8 +2,23 @@
         xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:pt="http://propertytools.org/wpf"
-        Title="PropertyGrid" Height="400" Width="300">
+        Title="PropertyGrid"
+        Width="650"
+        Height="400">
     <Grid>
-        <pt:PropertyGrid Margin="8" SelectedObject="{Binding SelectedPerson}"/>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+        <pt:PropertyGrid Grid.Column="0"
+                         Margin="8"
+                         SelectedObject="{Binding SelectedPerson}" />
+        <pt:PropertyGrid Grid.Column="1"
+                         Margin="8"
+                         SelectedObject="{Binding SelectedPerson}" >
+            <pt:PropertyGrid.PropertyItemFactory>
+                <pt:MetadataPropertyItemFactory />

```

Comment:
this demo project is supposed to be very simple... 
Please move the MetadataPropertyItemFactory example into the CustomFactoryDemo.
I suggest to add a tab page to the main window there